generate-stackbrew-pr.sh

```diff
@@ -3,14 +3,14 @@
 set -e
 . functions.sh
 
-if [ -z "$1" ]; then
-  COMMIT_ID="$TRAVIS_COMMIT"
-  COMMIT_MESSAGE="$TRAVIS_COMMIT_MESSAGE"
-  BRANCH_NAME="travis-$TRAVIS_BUILD_ID"
+if [ -z "${1}" ]; then
+  COMMIT_ID="${TRAVIS_COMMIT}"
+  COMMIT_MESSAGE="${TRAVIS_COMMIT_MESSAGE}"
+  BRANCH_NAME="travis-${TRAVIS_BUILD_ID}"
   GITHUB_USERNAME="nodejs-github-bot"
 else
-  COMMIT_ID="$1"
-  COMMIT_MESSAGE="$(git show -s --format=%B "$1")"
+  COMMIT_ID="${1}"
+  COMMIT_MESSAGE="$(git show -s --format=%B "${COMMIT_ID}")"
```

Comment:
this is actually not a style fix

Reply:
It's not strictly style related, but it does improve readability so I'm ok with it

Reply:
Stylewise, we should always assign numbered arguments and use that variable for better readability

Reply:
It's more like a bug then a style thing.

Reply:
Why is it a bug? Before and after behaves the exact same way. This is strictly for readability

Reply:
Or a bad smell may be more precise? Assigned a variable but never be referenced before. Anyway I'm not blocking this change but would like to have a discussion.

Reply:
Do you mean adding a `local COMMIT_ID`?

Reply:
I mean the combination of adding `COMMIT_ID="${1}"` and changing `git show -s --format=%B "$1"` to `git show -s --format=%B "${COMMIT_ID}"` looks more like the style thing, but the second change actually comes from a miss, so I won't call it a style change, personally.